src/extension/src/file_handlers/ExtConfigSettingsHandler.py

```diff
@@ -115,6 +115,7 @@ def read_file(self, seq_no):
             file_name = str(seq_no) + self.file_ext
             config_settings_json = self.json_file_handler.get_json_file_content(file_name, self.config_folder, raise_if_not_found=True)
             if config_settings_json is not None and self.are_config_settings_valid(config_settings_json):
+                cloud_type = self.get_ext_config_value_safely(config_settings_json, self.public_settings_all_keys.cloud_type)
```

Comment:
Do we need to update swagger for this?

Reply:
It originates within the respective services (CRP and HCRP) - I'd made the change in CRP to reflect in the extension input. It's not user input (no API, so no swagger).